runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesConfig.java

```diff
@@ -47,11 +47,22 @@ default boolean enableTooltips()
 		return true;
 	}
 
+	@ConfigItem(
+			keyName = "enableSkillColors",
+			name = "Enable skill colors",
+			description = "Use colors matching to the skill",
+			position = 1
+	)
+	default boolean enableSkillColors()
```

Comment:
Why is this even toggle?

Reply:
I didn't want to remove the option to set a value for all orbs.
With this change it's default on skill colors but you can uncheck the option and then set 
a value yourself.